src/styles/Icons.js

```diff
@@ -0,0 +1,3 @@
+import { Platform } from 'react-native';
+
+export const weatherIconsFontFamily = (Platform.OS === 'android') ?  'weathericons' : 'Weather Icons';
```

Comment:
did not try, it should work according to the doc. This icon is too  small and I don't really wanna create two files for it ATM